tests/functional/fake-firefox-binary.js

```diff
@@ -22,7 +22,19 @@ function toRDP(msg) {
   return [data.length, ':', data].join('');
 }
 
-// Start a TCP Server
+// Getting PORT from arguments
+function getPortFromArgs() {
+  const index = process.argv.indexOf('-start-debugger-server');
+  if (index === -1) {
+    throw new Error('The -start-debugger-server paramter is not present.');
```

Comment:
Nit, typo, paramter => parameter.

maybe also `is not present` => `is missing`